.ci/dockerfiles/Dockerfile.rhel8.6.release

```diff
@@ -0,0 +1,27 @@
+ARG HARBOR_URL=harbor.mellanox.com
+FROM $HARBOR_URL/hpcx/x86_64/rhel8.6/core:latest
+ARG _UID=6213
+ARG _GID=101
+ARG _LOGIN=swx-jenkins
+ARG _HOME=/var/home/$_LOGIN
+
+RUN sed -i 's/mirrorlist/#mirrorlist/;s!#baseurl=http://mirror.centos.org!baseurl=http://vault.centos.org!' /etc/yum.repos.d/* && \
+    echo "[mlnx-opt]" > /etc/yum.repos.d/mlnx-opt.repo && \
+    echo "name=RHEL 8.6 mirror" >> /etc/yum.repos.d/mlnx-opt.repo && \
+    echo "baseurl=http://webrepo.mtr.labs.mlnx/RH/optional/8.6/x86_64/" >> /etc/yum.repos.d/mlnx-opt.repo && \
+    echo "enabled=1" >> /etc/yum.repos.d/mlnx-opt.repo && \
+    echo "gpgcheck=0" >> /etc/yum.repos.d/mlnx-opt.repo && \
+    yum makecache
+
+RUN echo "${_LOGIN} ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
+    echo "root ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
+    mkdir -p ${_HOME} && \
+    groupadd -f -g "$_GID" "$_LOGIN" && \
+    useradd -u "$_UID" -g "$_GID" -s /bin/bash -m -d ${_HOME} "${_LOGIN}" && \
+    chown -R ${_LOGIN} ${_HOME}
+
+RUN yum install --allowerasing -y \
```

Comment:
I like to put list of programs yum (or any othe manager) installs 1 per line and in alphabetical order
It much easer to read an to maintain
For example here it'll be 
```
RUN yum install --allowerasing -y \
  autoconf \
  automake \
  gcc \
  gcc-c++ \
  git \
  libibverbs-devel \
  libnl3-devel \
  librdmacm \
  libtool \
  make \
  patch \
  rdma-core \
  rpm-build \
  sudo \
  unzip \
  wget
```